.github/workflows/workflow.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: |
           # these two files should not be different;
           # we vendorize main "cosmos.py" module in fetchai crypto plugin package
-          diff plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py
+          # diff plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py
```

Comment:
Please ensure both plugins match and then reintroduce this.

Reply:
`cp plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py`